### PR TITLE
show audit certificates deadline in question F 4.1 in all forms

### PR DESCRIPTION
**Pull request by macool**
also expose `audit_certificates_deadline` as application helper to pull & cache deadline from DB

### CODE REVIEW

**Comment by macool** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;F1znqF37&#x2F;922-qae16imp-change-question-to-pull-in-deadline-date-on-all-forms"> [QAE16IMP] Change question to pull in deadline date on all forms</a>

File: app/views/qae_form/_confirm_question.html.slim [hunk 1]
```diff
@@ -1,4 +1,4 @@
 label.selectable.selectable-confirm
   input.js-trigger-autosave type="checkbox" name=question.input_name checked=!!question.input_value *possible_read_only_ops
   - if question.text
-    == question.text 
+    == question.text.gsub("[AUDIT_CERTIFICATES_DEADLINE]", audit_certificates_deadline.decorate.formatted_trigger_time)
```
Comment (rusllonrails) [inline app/views/qae_form/_confirm_question.html.slim]: @macool 
```
== question.text.gsub("[AUDIT_CERTIFICATES_DEADLINE]", audit_certificates_deadline.decorate.formatted_trigger_time)
```
I guess would be good to do not run .gsub for all ConfirmQuestions.
And just run this replace for target question only.
```
- if question.key == :shortlisted_case_confirmation
  == question.text.gsub("[AUDIT_CERTIFICATES_DEADLINE]", audit_certificates_deadline.decorate.formatted_trigger_time)
- else
  == question.text
```

**Comment by rusllonrails** [comment]
Looks good :+1: 
Added small note above ^

**Comment by macool** [comment]
@rusllonrails i've amended, thanks

**Pull request merged**